health/notifications/pushover/metadata.yaml

```diff
@@ -0,0 +1,76 @@
+# yamllint disable rule:line-length
+---
+- id: 'notify-pushover'
+  meta:
+    name: 'PushOver'
+    link: 'https://pushover.net/'
+    categories:
+      - notify.agent
+    icon_filename: 'pushover.png'
+  keywords:
+    - PushOver
+  overview:
+    notification_description: |
+      Send notification to Pushover using Netdata's Agent alert notification feature, which supports dozens of endpoints, user roles, and more.
+      
```

Comment:
it looks like a pattern for `notification_description`, please , take a look on this file too.